src/Templates/src/templates/maui-blazor/global.json

```diff
@@ -0,0 +1,6 @@
+{
+  "sdk": {
+    "version": "6.0.200",
+    "rollForward": "latestMinor"
```

Comment:
Just a sanity check, @jonathanpeppers there will be no prerelease sdk versions, right? so we don't need to set that here to allow?.

Reply:
You might have 6.0.300-preview1, yeah we should add that.

I'll update that after we can get past the iOS/`@(Content)` issue.